contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -367,12 +389,18 @@ create_bbf_db_internal(const char *dbname, List *options, const char *owner, int
 	dbo_role = get_dbo_role_name(dbname);
 	db_owner_role = get_db_owner_name(dbname);
 	guest = get_guest_role_name(dbname);
+	guest_scm = get_guest_schema_name(dbname);
 
 	if (SearchSysCacheExists1(NAMESPACENAME, PointerGetDatum(dbo_scm)))
 		ereport(NOTICE,
 				(errcode(ERRCODE_DUPLICATE_SCHEMA),
 				 errmsg("schema \"%s\" already exists, skipping", dbo_scm)));
 
+	if (SearchSysCacheExists1(NAMESPACENAME, PointerGetDatum(guest_scm)))
+		ereport(ERROR,
```

Comment:
It should be an error if the schema already exists before the create database. We do not want to continue when that happens. I am not sure why the first one is a NOTICE instead of an error. The next couple of checks are all ERRORs and I think ERROR is. the correct behaviour here.